### PR TITLE
Feat/alert/UI: AlertScreen UI

### PR DESCRIPTION
**Pull request by agonzalez-r**
## Overview
The `AlertScreen` is where users can customise their alert and submit it so that the other users will receive it. They will then be displayed in the `AlertListScreen` #23. 

Screenshot 2024-10-10 at 19 10 21

## Key Features
- Dropdown menu with the period product option the user prefers
- Dropdown menu with the urgency levels 
- Followed the Figma, but I think there is room for improvement for the design

## Future improvements
- An Alert will be created and store in the repository once the model and viewModel are finished 
- I created the data class for the Alerts in the branch `Feat/alert/model` #17, all necessary information will be provided in this screen in order to effectively create an instance
-  **For the moment the Location is just a string**, next sprint we will implement a Location data class with coordinates so that they can be displayed in a map 
- UI-wise, I think this screen can be improved, maybe do a round button and better arrange the entry components

## Tests
- Verified that all components are correctly displayed and the interaction is correct

### CODE REVIEW

**Review by lazarinibruno — DISMISSED** [review]
Thanks for your work! 

## Comments

* The inclusion of tests verifying that everything is correctly displayed is top! 
* The UI covers the basic features that the screen will have. 
* I like how you modularized the code by creating your own custom `ExposedDropDownMenuSample` composable.

## Improvements 

As you mentioned, the UI could be improved to make it feel more modern-looking. Some improvements could be:

* The explanation at the top of the screen could be shortened or even omitted.
* Having a button that upon clicking on it, cycles through the menstrual products.
* The same for the urgency level

## Steps before merge

LGTM

File: app/src/main/java/com/android/periodpals/ui/alert/AlertScreen.kt [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  Scaffold(
+      modifier = Modifier.testTag("alertScreen"),
+      content = { paddingValues ->
+        Column(
+            modifier = Modifier.fillMaxSize().padding(16.dp).padding(paddingValues),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.SpaceEvenly) {
+              // Text Instruction
+              Text(
+                  "Push a notification that will be received by users around you so that they can help you find the period product that you need",
```
Comment (charliemangano) [inline app/src/main/java/com/android/periodpals/ui/alert/AlertScreen.kt:48]:
```suggestion
                  "Push a notification to users near you! If they are available and have the products you need, they'll be able to help you!"
```

**Review by charliemangano — APPROVED** [review]
Nice work!

### Comments
 - Sonarcloud is not correctly configured (see #26), you have an amazing 93.2% line coverage, thank you for your work!
 - The tests should be separated into multiple test functions for maintainability and readability.


### Improvements
 - Make the dropdown menus and the text field the same width.
 - Once we have icons designed for the period products and alert levels, incorporate those into the dropdown menu items.


### Steps before merge
lgtm

**Comment by agonzalez-r** [comment]
@charliemangano thanks for the feedback, you're right the dropdown menus and text fields should have the same width. 

However I tried modifying the width with a `.fillMaxWidth()` in the `Modifier` of `ExposedDropdownMenuBox()`, and it doesn't change the width at all. 

I also tried to directly modifying by adding `.width(500.dp)`. It does absolutely nothing, I wonder if the composable `ExposedDropdownMenuBox()` has a fixed size. 

Otherwise I could just adjust the width of the text fields to be narrower and match the menus, but it would look weird in my opinion.

**Review by lazarinibruno — APPROVED** [review]
LGMT

**Review by charliemangano — APPROVED** [review]
lgtm!
thanks for the changes

**Pull request merged**